app/grandchallenge/cases/migrations/0013_rawimageuploadsession_reader_study.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.2.4 on 2019-08-14 13:27
+# Generated by Django 2.2.4 on 2019-08-14 14:02
```

Comment:
Was this re-generated??!

Reply:
Yeah - conflicting changes to models in both of the branches that we want to move to master so needed to be removed to run `makemigrations` again.